src/test-geogram-install2.cpp

```diff
@@ -0,0 +1,7 @@
+#include <iostream>
+#include <geogram/mesh/mesh.h>
+
+void toto_mesh()
+{
+    std::cout << GEO::MeshCellDescriptors::tet_descriptor.nb_facets << std::endl;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this usage of "::std::cout" by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3AS106)